### ISSUE TITLE
SyntaxError when running foliadocserve

### ISSUE THREAD

**Issue by matjemeisje** [proycon/foliadocserve#7]
I installed version 0.5.1 through `pip install foliadocserve` (with sudo). Running `foliadocserve -h` or `foliadocserve -d /path/to/docs` returns an error:

`~$ foliadocserve -h`
`Traceback (most recent call last):`
`  File "/usr/local/bin/foliadocserve", line 7, in <module>`
`    from foliadocserve.foliadocserve import main`
`  File "/usr/local/lib/python2.7/dist-packages/foliadocserve/foliadocserve.py", line 701`
`    def save(self, *args, message=""):`
`                                ^`
`SyntaxError: invalid syntax`

Tried it on two separate Linux machines.

**Comment by proycon** [comment]
Ah, you have to use Python 3, not Python 2.7 (I'm surprised you managed to get it installed for Python 2.7 without error even). On Debian/Ubuntu systems, use ``pip3`` (package ``python3-pip``) and ``python3``.

**Comment by matjemeisje** [comment]
Then it seems your manual is incomplete.

**Comment by matjemeisje** [comment]
OK, it took me another 30 mins, but I got it working. First, I uninstalled everything I had installed with pip. Then installed pip3 through aptitude, reinstalled foliadocserve with pip3, still didn't work.
Some googling told me I had to `sudo easy_install3 -U pip` to get pip working correctly with python3. That gave me an extra command `pip3.5` which I could then use to successfully install foliadocserve.
